components/Formatter.php

```diff
@@ -135,4 +142,57 @@ private function replaceCommentHeadlines($html)
     {
         return preg_replace('/<h\d+.*?>(.*?)<\\/h\d+>/i',"<p><strong>\\1</strong></p>", $html);
     }
+
+
+    /**
+     * @param string $html
+     *
+     * @return string
+     */
+    private function replaceImageUrlForProxy($html)
+    {
+        if (!isset(Yii::$app->params['image-proxy'], Yii::$app->params['image-proxy-secret'])) {
+            return $html;
+        }
+
+        return preg_replace_callback('/(<img[^>]+?)src=(["\'])([^"\']+)\2/i', function($matches) {
```

Comment:
src could be specified without " or '

Reply:
`<img(.*?)src=(['"]?)(.*?)(['" >])`?

Reply:
- part after `<img` should be excluding `>` in case there is an image tag without src before another image tag with src, it will match wrong.
- about `(.*?)(['" >])` I'd like to avoid trying to guess urls on broken image tags. these are expected to not work and should be blocked by Content-Security-Policy. To avoid attacks like https://githubengineering.com/githubs-post-csp-journey/